shell/browser/notifications/win/windows_toast_notification.cc

```diff
@@ -146,7 +146,7 @@ void WindowsToastNotification::Remove() {
     return;
 
   ScopedHString group(kGroup);
-  ScopedHString tag(base::as_wcstr(base::UTF8ToUTF16(notification_id())));
+  ScopedHString tag(base::as_wcstr(base::UTF8ToUTF16(std::to_string(std::hash<std::string>{}(notification_id())))));
```

Comment:
Could we extract this to a helper function in the anonymous namespace alongside a comment about why it's necessary? As it stands now I think it'd be a bit confusing for future reviewers/readers of this code.

Reply:
Sure. It's a good idea.

Reply:
@codebytere 
I optimized the changes base on your suggestions.
Chromium also handles the Tag property by calculating hash.